pkg/kubelet/image_manager.go

```diff
@@ -269,7 +269,7 @@ func (im *realImageManager) freeSpace(bytesToFree int64, freeTime time.Time) (in
 	sort.Sort(byLastUsedAndDetected(images))
 
 	// Delete unused images until we've freed up enough space.
-	var lastErr error
+	deletionErrors := make(map[string]error)
```

Comment:
There is a pkg/util/errors[https://github.com/kubernetes/kubernetes/blob/master/pkg/util/errors/errors.go] package if you are interested in aggregating al errors as a single error.